geoips/commandline/args.py

```diff
@@ -638,6 +638,26 @@ def add_args(parser, arglist=None, legacy=False):
                     be passed to the algorithm in full.""",
         )
 
+    if arglist is None or "output_checker_name" in arglist:
+        procflow_group.add_argument(
+            "--output_checker_name",
+            default=None,
+            help="""Output Checker Name Override.
+                    - By default, GeoIPS determines the appropriate
+                      output_checker plugin to use for comparisons based on
+                      the file extension of the output.
+                    - If a particular file extension is not explicitly
+                      supported, but is expected to be able to utilize one of
+                      the existing output checkers (e.g. using the "text"
+                      output checker with a CSV file), this argument allows the
+                      name of the checker to be passed in to override the default.
+                    - I.e., to force the use of the "text" ouput checker with a
+                      product that produces a CSV output output_checker plugin,
+                      you would pass:
+                        --output_checker_name text
+                    """,
```

Comment:
You can now specify an explicit output checker name instead of relying on geoips auto-identifying it!